src/common/Input/Input.js

```diff
@@ -368,6 +369,7 @@ Input.propTypes = {
   focused: PropTypes.bool,
   iconClass: PropTypes.string,
   iconOnClick: PropTypes.func,
+  id: PropTypes.string,
```

Comment:
There should be a default value for not required props

Reply:
So this statement will never be false. ` data-testid={id ? `${id}-input` : 'input'}`

Reply:
You can have an empty string as a default value